project/SequentialTestExecutionInCI.scala

```diff
@@ -0,0 +1,18 @@
+import sbt.Keys._
+import sbt._
+
+/**
+ * This plugin disables parallel execution in travis
+ */
+object SequentialTestExecutionInCI extends AutoPlugin {
```

Comment:
This is interesting.  One (completely not researched) theory I have is that the number of containers running in the Travis VM is causing a lot of swapping, and thus slowdown.  testcontainers is one solution because we can manage container lifecycle per test.  Another would be to split out tests into separate travis jobs.